Vagrantfile

```diff
@@ -114,11 +115,11 @@ ansible_provision = proc do |ansible|
     # Use monitor_address_block instead of monitor_interface:
     ansible.extra_vars.delete(:monitor_interface)
     ansible.extra_vars = ansible.extra_vars.merge({
-      cluster_network: "#{SUBNET}.0/16",
+      cluster_network: "#{CLUSTER_SUBNET}.0/16",
```

Comment:
I've updated the PR to use the same private subnet 192.168.0.0/16 for both public and cluster networks. Linode, and OpenStack for that matter, just have a slightly different network configuration.